pkg/make_deb.py

```diff
@@ -31,7 +31,7 @@
 
 # list of debian fields : (name, mandatory, wrap[, default])
 # see http://www.debian.org/doc/debian-policy/ch-controlfields.html
-from helpers import GetFlagValue
+from rules_pkg.private.helpers import GetFlagValue
```

Comment:
Is the `rules_pkg` part derived from the workspace name?

This would make it more difficult for users to change the workspace name on import.  Perhaps `imports = [".."]` could be added to the corresponding `py_library`s?  Modules could then be imported relative to `private`. This may end up being a largely moot point when all the `make_*` scripts are moved to `pkg/private`, though.

Speaking of changing the workspace name, I found that in order to do this, we need to rename all uses of the `rpmbuild` toolchain.  Not sure if there's anything we want to do or reasonably can do about this.

Reply:
I removed the rules_pkg part.